src/server.cpp

```diff
@@ -1958,6 +1976,30 @@ void Server::SendMovePlayer(u16 peer_id)
 	Send(&pkt);
 }
 
+void Server::SendKnockBack(const u16 peer_id, const v3f &direction, const f32 &time_knockback, const u16 id_player_knockback)
```

Comment:
This makes zero difference because
* this is a function argument
* it can be inferred anyway